test-cypress/cypress/commands/server.ts

```diff
@@ -0,0 +1,85 @@
+// load the global Cypress types
+/// <reference types="cypress" />
+export const DEFAULT_DATE = new Date(2021, 1, 16, 22);
+import url = require("url");
```

Comment:
Mixing `require` with `import` seems weird - shouldn't this be `import url from 'url'`, or `import * as url from 'url'` if url has no default export?